Vagrantfile

```diff
@@ -148,8 +125,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     config.vm.provider "lxc" do |lxc, override|
-        override.vm.box = custom_config.get('lxc_box_name', 'jessie64-lxc')
-        box_url = custom_config.get('lxc_box_url', 'http://vagrantbox-public.liip.ch/liip-jessie64-lxc.box')
+        override.vm.box = custom_config.get('lxc_box_name', 'drifter/jessie64-base')
```

Comment:
not sure you need to override the box config if it's the same as the "normal" one since all info are in the json.

I'll test without :P